package.json

```diff
@@ -51,7 +51,7 @@
     "build:web": "nx build @novu/web",
     "build:widget": "nx build @novu/widget",
     "build:embed": "nx build @novu/embed",
-    "build:storybook": "nx run @novu/web:build-storybook",
+    "build:storybook": "nx run @novu/design-system:build-storybook",
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** Not necessarily related to this change, but it would be great to have storybook deploy-previews in web as well in the future.‏